modular/gater/request_context.go

```diff
@@ -185,11 +215,7 @@ func (r *RequestContext) verifySignatureV1(requestSignature string) (sdk.AccAddr
 	}
```

Comment:
I think we dont have to do `VerifySignature` after the addr is `RecoverAddr`

Reply:
Done